_dashboards/get-started/dql.md

```diff
@@ -0,0 +1,165 @@
+```
+
+DQL supports leading and trailing wildcards, so you can search for any terms that match your pattern.
+
+```
+host.keyword:*.example.com/*
+```
+
+To check if a field exists or has any data, use a wildcard to see if Dashboards returns any results.
+
+```
+host.keyword:*
+```
+
+## Searching with Boolean queries
+
+To mix and match or combine multiple queries for more refined results, you can use the boolean operators `and`, `or`, and `not`. DQL is not case sensitive, so `AND` and `and` are the same.
+
+```
+host.keyword:www.example.com and response.keyword:200
+```
+
+The following example shows how to use multiple operators in one query.
+
+```
+geo.dest:US or response.keyword:200 and host.keyword:www.example.com
+```
+
+Remember that boolean operators follow the logical precedence order of `not`, `and`, and `or`, so if you have an expression like the previous example, `response.keyword:200 and host.keyword:www.example.com` gets evaluated first.
+
+To avoid confusion, use parentheses to dictate the order in which you want to evaluate operands. If you want to evaluate `geo.dest:US or response.keyword:200` first, the expression is:
+
+```
+(geo.dest:US or response.keyword:200) and host.keyword:www.example.com
+```
+
+## Querying dates and ranges
+
+DQL supports numeric inequalities.
+
+```
+bytes >= 15 and memory < 15
+```
+
+Similarly, you can use the same method to find a date before or after the query. `>` indicates a search for a date after the specified date, and `<` returns dates before.
+
+```
+@timestamp > "2020-12-14T09:35:33"
+```
+
+## Querying nested fields
+
+Searching a document with nested fields requires you to specify which parts of the document to retrieve. In this example, `superheroes` is the nested field type.
+
+```json
+{
+ "superheroes":[
+    {
+      "hero-name": "Superman",
+      "real-identity": "Clark Kent",
+      "age": 28
+    },
+    {
+      "hero-name": "Batman",
+      "real-identity": "Bruce Wayne",
+      "age": 26
+    },
+    {
+      "hero-name": "Flash",
+      "real-identity": "Barry Allen",
+      "age": 28
+    },
+    {
+      "hero-name": "Robin",
+      "real-identity": "Dick Grayson",
+      "age": 15
+    }
+  ]
+}
+```
+
+To retrieve a specific field using DQL, use the notation:
+
+```
+superheroes: {hero-name: Superman}
+```
+
+To retrieve multiple objects from your document, specify all the fields you want to retrieve, as shown in the following notation:
+
+```
+superheroes: {hero-name: Superman} and superheroes: {hero-name: Batman}
+```
+
+The previous boolean and range queries still work, so you can submit a more refined query, as shown in the following notation:
+
+```
+superheroes: {hero-name: Superman and age < 50}
+```
+
+## Querying double nested objects 
+
+If a document has double nested objects (object nested inside another object), retrieve data by specifying all the levels. In the following example document, `superheroes` is double nested.
+
+
+
+```json
+{
```

Comment:
This should be :


{
  "justice-league": [
    {
      "superheroes":[
    {
      "hero-name": "Superman",
      "real-identity": "Clark Kent",
      "age": 28
    },
    {
      "hero-name": "Batman",
      "real-identity": "Bruce Wayne",
      "age": 26
    },
    {
      "hero-name": "Flash",
      "real-identity": "Barry Allen",
      "age": 28
    },
    {
      "hero-name": "Robin",
      "real-identity": "Dick Grayson",
      "age": 15
    }
  ]
}
  ]
}